roles/cms/tasks/main.yml

```diff
@@ -15,6 +15,8 @@
     - libffi-dev
     - build-essential # Required to build some CMS Python dependencies
     - libcups2-dev # Required by pycups CMS Python dependency
+    - pkg-config
+    - libsystemd-dev
```

Comment:
This was needed for Isolate, but if you use the packaged version, it's no longer useful.